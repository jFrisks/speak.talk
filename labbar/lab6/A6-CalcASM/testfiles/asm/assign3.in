int main(){
  int x;
  x = f(3 - 2);
  int y = g(3, 42);
  print(x);
  print(y);
}

int f(int a){
  print(a);
  int y = g(3, a);
  return y;

}

int g(int a, int b){
  int x = a + b;
  return a * b * x;
}