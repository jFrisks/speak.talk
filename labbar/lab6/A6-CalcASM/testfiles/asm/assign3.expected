1
12
3780